cms/jinja2/templates/components/streamfield/release_content_section.html

```diff
@@ -3,14 +3,16 @@ <h2>{{ value.title }}</h2>
 
     <ul class="ons-list ons-list--bare">
         {% for item in value.links %}
-            <li class="ons-list__item">
-                {% if item.link %}
-                    <p><a href="{{ item.link.url }}">{{ item.link.text }}</a></p>
-                    {% if item.link.description %}
-                        <p>{{ item.link.description }}</p>
-                    {% endif %}
-                {% endif %}
-            </li>
+            {% if item.link %}
+                <li class="ons-list__item">
+                    <div class="ons-u-mb-s">
+                        <a href="{{ item.link.url }}">{{ item.link.text }}</a>
+                        {% if item.link.description %}
+                            {{ item.link.description|richtext }}
+                        {% endif %}
+                    </div>
+                </li>
+            {% endif %}
```

Comment:
These changes LGTM as they are.

I was cross-referencing this with the `onsList` component (well, DS macro) and it looks like it adds specific classes on the `a` tag:

- https://github.com/ONSdigital/design-system/blob/main/src/components/list/_macro.njk#L88 for internal links
- https://github.com/ONSdigital/design-system/blob/main/src/components/external-link/_macro.njk#L4 for external links

which I think are worth following. What do you think?

Reply:
on a second thought - let's do this as a follow up task (can you add it as a task to the statistical article or methodology DS integration ticket)?

Reply:
I'll resolve this conversation as there is nothing further for @zerolab to check - I have added a comment on the statistical article page ticket.